data/json/itemgroups/furniture.json

```diff
@@ -2,46 +2,71 @@
   {
     "type": "item_group",
     "id": "knifeblock",
-    "items": [ [ "knife_steak", 85 ], [ "knife_butcher", 10 ], [ "kukri", 1 ], [ "tongs", 28 ], [ "meat", 1 ] ]
+    "subtype": "distribution",
+    "entries": [
+      { "item": "knife_steak", "prob": 85 },
+      { "item": "knife_butcher", "prob": 10 },
+      { "item": "kukri", "prob": 1 },
+      { "item": "tongs", "prob": 28 },
+      { "item": "meat", "prob": 1 }
+    ]
   },
   {
     "type": "item_group",
     "id": "oven",
-    "items": [
-      [ "pan", 80 ],
-      [ "rolling_pin", 25 ],
-      [ "waffleiron", 25 ],
-      [ "teapot", 25 ],
-      [ "pot", 80 ],
-      [ "pot_copper", 20 ],
-      [ "pot_canning", 15 ],
-      [ "pressure_cooker", 5 ]
+    "subtype": "distribution",
+    "entries": [
+      { "item": "pan", "prob": 80 },
+      { "item": "rolling_pin", "prob": 25 },
+      { "item": "waffleiron", "prob": 25 },
+      { "item": "teapot", "prob": 25 },
+      { "item": "pot", "prob": 80 },
+      { "item": "pot_copper", "prob": 20 },
+      { "item": "pot_canning", "prob": 15 },
+      { "item": "pressure_cooker", "prob": 5 }
     ]
   },
   {
     "type": "item_group",
     "id": "bed",
-    "items": [ [ "blanket", 25 ], [ "down_blanket", 13 ], [ "pillow", 30 ], [ "down_pillow", 15 ], [ "electric_blanket", 4 ] ]
+    "subtype": "distribution",
+    "entries": [
+      { "item": "blanket", "prob": 25 },
+      { "item": "down_blanket", "prob": 13 },
+      { "item": "pillow", "prob": 30 },
+      { "item": "down_pillow", "prob": 15 },
+      { "item": "electric_blanket", "prob": 4 }
+    ]
   },
   {
     "type": "item_group",
     "id": "pool_table",
-    "items": [ [ "pool_cue", 4 ], [ "pool_ball", 40 ] ]
+    "name": {  },
```

Comment:
This is causing the tests to fail.  Is this supposed to be here?

Reply:
I removed it, thank you